mbtools/utils.py

```diff
@@ -54,3 +54,14 @@ def parse_question_bank_for_html(mbz_dir, ids=None):
         for item in question.html_elements():
             html.append(item)
     return html
+
+
+def find_references_containing(tree, src_content):
```

Comment:
Minor: Instead of `tree` would it be more clear if we named this argument `content_etree`?